README.md

```diff
@@ -39,3 +39,253 @@ Default setup for Ditto uses Ngix with basic authentication. A custom authentica
+
+---
+
+### Create Policy
+
+#### `ditto-client policy create <policy_id> <policy_file>`
+Create a new policy.
+
+```bash
+# Create a new policy
+ditto-client policy create "my.namespace:new-policy" examples/cli-examples/policy.json
+```
+
+#### `ditto-client policy get <policy_id>`
+Retrieve a specific policy by ID.
+
+```bash
+# Get a policy
+ditto-client policy get "my.namespace:my-policy"
+```
+
+#### `ditto-client policy entries <policy_id>`
+List policy entries.
+
+```bash
+# List all policy entries
+ditto-client policy entries "my.namespace:my-policy"
+```
+
+---
+
+### Things Management
+
+#### `ditto-client things create <thing_id> <data_file>`
+Create a new thing.
+
+```bash
+# Create a new thing
+ditto-client things create "my.namespace:new-thing" examples/cli-examples/thing.json
+```
+
+#### `ditto-client things list`
+List all things with optional filtering.
+
+```bash
+# List all things
+ditto-client things list
+
+# List things with specific fields
+ditto-client things list --fields "thingId,attributes"
+
+# List specific things by ID
+ditto-client things list --ids "my.namespace:new-thing"
+```
+
+#### `ditto-client things get <thing_id>`
+Retrieve a specific thing by ID.
+
+```bash
+# Get a specific thing
+ditto-client things get "my.namespace:my-thing"
+```
+
+#### `ditto-client things update <thing_id> <patch_file>`
+Update a thing using JSON patch.
+
+```bash
+# Update a thing
+ditto-client things update "my.namespace:my-thing" examples/cli-examples/thing.json
+```
+
+---
+
+### Search Operations
+
+#### `ditto-client search query`
+Search for things using RQL (Resource Query Language).
+
+```bash
+# Search all things
+ditto-client search query
+
+# Search with filter
+ditto-client search query --filter 'eq(attributes/location,"Kitchen")'
+
+# Search with size limit and sorting
+ditto-client search query --option "size(3),sort(+thingId)"
+
+# Search in specific namespaces
+ditto-client search query --namespaces "my.namespace"
+```
+
+#### `ditto-client search count`
+Count things matching search criteria.
+
+```bash
+# Count all things
+ditto-client search count
+
+# Count things with filter
+ditto-client search count --filter 'eq(attributes/location,"Kitchen")'
+```
+
+---
+
+### Connection Management (DevOps)
+
+#### `ditto-client connection create <connection_id>`
+Create a new connection.
+
+```bash
+# Create a connection
+ditto-client connection create "new-connection" --definition "connection-config"
+```
+
+#### `ditto-client connection list`
+List all connections.
+
+```bash
+# List all connections
+ditto-client connection list
+
+# List with specific fields
+ditto-client connection list --fields "id,connectionStatus"
+```
+
+#### `ditto-client connection get <connection_id>`
+Retrieve a specific connection by ID.
+
+```bash
+# Get a connection
+ditto-client connection get "my-connection"
+
+# Get with specific fields
+ditto-client connection get "my-connection" --fields "id,status"
+```
+
+---
+
+### **CLEAN UP**
```

Comment:
The delete commands should be their respective section with THings, policies etc ... no need to make separate section